src/fabric.c

```diff
@@ -153,15 +153,15 @@ static struct ofi_prov *ofi_create_prov_entry(const char *prov_name)
  */
 static void ofi_ordered_provs_init(void)
 {
-	char *ordered_prov_names[] =
-			{"psm2", "psm", "usnic", "mlx", "verbs","gni",
-			 "bgq", "netdir", "ofi_rxm", "ofi_rxd",
-			/* Initialize the socket(s) provider last.  This will result in
-			 * it being the least preferred provider. */
-
-			/* Before you add ANYTHING here, read the comment above!!! */
-			"UDP", "sockets", "tcp" /* NOTHING GOES HERE! */};
-			/* Seriously, read it! */
+	char *ordered_prov_names[] = {
+		"psm2", "psm", "usnic", "mlx", "ofi_rxm", "verbs",
+		"gni", "bgq", "netdir", "ofi_rxd",
```

Comment:
Please change the order to: psm2, psm, usnic, mlx, gni, bgq, ofi_rxm, verbs, netdir ...  Basically, move verbs down, so that rxm doesn't end up earlier than gni or bgq.  (I don't know if those providers support MSG EPs currently, but they do have their own RDM EP support.

Reply:
Is there a reason why the ofi_rxm + verbs provider is being preferred over the native verbs provider? The native verbs provider has its own implementation of RDM endpoints.

Reply:
RxM over Verbs is more stable with better performance in most cases, plus supports more features.  The verbs RDM support will be removed after the 1.6 release.  All development focus has been on RxM.

Reply:
Where was the removal of verbs RDM support for 1.6 discussed?

Reply:
> Where was the removal of verbs RDM support for 1.6 discussed?

This hasn't been discussed yet, we're planing this. FWIW, support of verbs/RDM is experimental (this says [fi_verbs man page](https://github.com/ofiwg/libfabric/blob/b0b1df91ed2153cb93f29daee3ff1c97ad85783d/man/man7/fi_verbs.7#L18))